client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/pattern-selector.tsx

```diff
@@ -94,6 +95,13 @@ const PatternSelector = ( {
 					</Delayed>
 				</div>
 			</div>
+			<div className="pattern-selector__footer">
```

Comment:
[nit] Maybe we have to check the `multiple` here. Otherwise, we will have the empty footer div when editing the selected pattern.